service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/EhrExtractStatusService.java

```diff
@@ -200,6 +215,25 @@ public EhrExtractStatus updateEhrExtractStatusAccessDocumentDocumentReferences(
         return ehrExtractStatus;
     }
 
+    public void updateEhrExtractStatusAcknowledgment(SendAcknowledgementTaskDefinition sendAcknowledgementTaskDefinition) {
```

Comment:
Small typo in the method name
```suggestion
    public void updateEhrExtractStatusAcknowledgement(SendAcknowledgementTaskDefinition sendAcknowledgementTaskDefinition) {
```